### DISCUSSION TITLE
Dynamic HLS Version (EXT-X-VERSION)

### DISCUSSION THREAD

**Discussion by mauricioabreu** [OvenMediaLabs/OvenMediaEngine#1718] [Ideas]
Some players, like Roku, can't play LL-HLS if the `EXT-X-VERSION` is 10. Chunklists are based on HLS v10.

Would you accept a pull request adding a configuration in the XML that allows users to choose a specific version?

**Comment by getroot** [comment]
LLHLS supports the _HLS_legacy=YES querystring. In this case, LLHLS will respond with EXT-X-VERSION as 6. (as of 0.17.1)

However, as far as I know, Roku may not support the m4s format. In that case, use HLSv3. Its EXT-X-VERSION is 3.

https://airensoft.gitbook.io/ovenmediaengine/streaming/hls
